src/OpenLoco/src/Ui/Windows/CompanyWindow.cpp

```diff
@@ -1779,10 +1781,10 @@ namespace OpenLoco::Ui::Windows::CompanyWindow
 
             if (company->id() == CompanyManager::getControllingId())
             {
-                self.widgets[widx::currentLoan].type = WidgetType::textbox;
-                self.widgets[widx::loan_decrease].type = WidgetType::button;
-                self.widgets[widx::loan_increase].type = WidgetType::button;
-                self.widgets[widx::loan_autopay].type = WidgetType::checkbox;
+                self.widgets[widx::currentLoan].hidden = false;
+                self.widgets[widx::loan_decrease].hidden = false;
+                self.widgets[widx::loan_increase].hidden = false;
+                self.widgets[widx::loan_autopay].hidden = false;
```

Comment:
Can be moved out of the `if` and assigned the bool `company->id() != CompanyManager::getControllingId()`.